src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/Wizard.vue

```diff
@@ -1,10 +1,10 @@
 <template>
   <slot name="header"></slot>
-  <v-stepper v-model="wizardStore.step" min-height="100dvh" :alt-labels="true" :mobile="$vuetify.display.mobile">
-    <slot name="stepperHeader">
+  <v-stepper v-model="wizardStore.step" min-height="100dvh" :alt-labels="true" :non-linear="true" :elevation="0">
```

Comment:
Confirming we do need this non-linear prop set?

Reply:
we do not. I removed it. Great catch!